packages/teleterm/src/services/logger/loggerService.ts

```diff
@@ -20,7 +20,7 @@ export default function createLoggerService(opts: Options): LoggerService {
         maxsize: 4194304, // 4 MB - max size of a single file
```

Comment:
TIL we actually limit the logs to some size, that's good to know. I was worried that we're going to leave hundreds of megabytes of logs on the drives of Connect users. ;)